core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -536,6 +552,8 @@ object ConfigCommand extends Logging {
           adminClient.listTopics(new ListTopicsOptions().listInternal(true)).names().get().asScala.toSeq
         case ConfigType.Broker | BrokerLoggerConfigType =>
           adminClient.describeCluster(new DescribeClusterOptions()).nodes().get().asScala.map(_.idString).toSeq :+ BrokerDefaultEntityName
+        case ConfigType.ClientMetrics =>
+          adminClient.describeCluster(new DescribeClusterOptions()).nodes().get().asScala.map(_.idString).toSeq
```

Comment:
I expect this needs to be `adminClient.describeConfigs`.

Reply:
I missed this, actually this is the place where if entity name is null then utility fetched all by issuing admin client API. I have added an exception here which we shall figure out when supporting describe all.